index.html

```diff
@@ -0,0 +1,115 @@
+                <li class="login-link"><a href="#"><i class="fa-brands fa-facebook fa-1.5x"></i></a></li>
+                <li class="login-link"><a href="#"><i class="fa-brands fa-twitter fa-1.5x"></i></a></li>
+                <li class="login-link"><a href="#">English</a></li>
+                <li class="login-link"><a href="#">My page</a></li>
+                <li class="login-link"><a href="#">og out</a></li>
+            </ul>
+        </div>
+        <nav class="navbar">
+            <div class="logo">
+                <img src="project-image/logo.jpg" alt="image">
+            </div>
+            <ul class="desktop-menu">
+                <li class="nav-item">
+                    <a href="about.html" class="nav-link active">About</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#program" class="nav-link">Services</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#join" class="nav-link">Instructors</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#sponsor" class="nav-link">Sponsor</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#news" class="nav-link">News</a>
+                </li>
+                <li class="nav-item">
+                    <p class="campaign">Apply Programs</p>
+                </li>
+            </ul>
+            
+            <div class="mobile-menu">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+        </nav>
+    </header>
+    
+    <main>
+        <section class="container1">
+            <section class="intro">
+                <p class="hello">"Hello! OPen to work "</p>
+                <h1 class="creative">Gateway is open for software development and training.  </h1>
+            </section>
+    
+            <section class="joy">
+                <p>We offer different services such as constructing, altering, repairing, or modifying garments for customers based on their specifications, needs, and preferences</p>
+                <p>Also we take our customers' measurements, assist in faric selection, and arrange fittings to determine whether additional adjustments are needed.</p>
+            </section>
+            <section class="date-place">
+                <p class="date">2023 (Fri) ~ (Sun)</p>
+                <p class="place"> @ Ethiopia,Addis ababa,megenagna,No.office:312 </p>
+            </section>
+        </section>
+        
+        <section class="main-prog-contents" id="program">
+            <h2 class="main-prog">Main Services</h2>
+            <div class="h-r1"></div>
+            <ul class="program-list"></ul>
+            <button type="button" class="join-btn">see our product here</button>
+            <a href="#" class="all-project">Apply for program</a>
+        </section>
```

Comment:
- Great job for making the service list dynamic, I think it would be better if you make the logo size equal and align the content straight to have better UI 👍 

#### Given Design 👇
<img width="661" alt="2023-08-10_11-51_1" src="https://github.com/abel12-pro/Capstone-1/assets/52010789/b14d6c2a-9e5f-48b5-b3c3-f64dcc5a47f7">

#### Your Design 👇
<img width="949" alt="2023-08-10_11-51" src="https://github.com/abel12-pro/Capstone-1/assets/52010789/fcf8c012-d205-450c-a128-e890f1d23840">